Unicode UI/MitSciUnicodeUI.pax

```diff
@@ -0,0 +1,1283 @@
+	yourself.
+
+package binaryGlobalNames: (Set new
+	yourself).
+
+package globalAliases: (Set new
+	yourself).
+
+package setPrerequisites: (IdentitySet new
+	add: '..\..\Core\Object Arts\Dolphin\Database\Database Connection Base';
+	add: '..\..\Core\Object Arts\Dolphin\IDE\Base\Development System';
+	add: '..\..\Core\Object Arts\Dolphin\Base\Dolphin';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Views\Common Controls\Dolphin Common Controls';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Models\List\Dolphin List Models';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Presenters\List\Dolphin List Presenter';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Base\Dolphin MVP Base';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Presenters\Prompters\Dolphin Prompter';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Presenters\Text\Dolphin Text Presenter';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Type Converters\Dolphin Type Converters';
+	add: '..\..\Core\Object Arts\Dolphin\MVP\Models\Value\Dolphin Value Models';
+	add: '..\..\Core\Contributions\Refactory\Refactoring Browser\Environments\RBEnvironments';
+	add: '..\..\Core\Object Arts\Dolphin\System\Compiler\Smalltalk Parser';
+	add: '..\Udo Schneider\Unicode\US Unicode';
+	yourself).
+
+package!
+
+"Class Definitions"!
+
+Object subclass: #MitSciUnicodeAPIScanner
+	instanceVariableNames: 'ansiAPIFunctionMethods'
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+ODBCLibrary subclass: #ODBCLibraryW
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+GDILibrary subclass: #GDILibraryW
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+UserLibrary subclass: #UserLibraryW
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+WNDCLASS subclass: #WNDCLASSW
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+MitSciUnicodeAPIScanner subclass: #MitSciUnicodeTransformer
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+Shell subclass: #TestUnicodeShell
+	instanceVariableNames: 'listPresenter textPresenter'
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+FlipperInspector subclass: #FlipperInspectorW
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+TestUnicodeShell subclass: #TestNonUnicodeShell
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+
+"Global Aliases"!
+
+
+"Loose Methods"!
+
+!ByteArray methodsFor!
+
+asUnicodeString
+	"Answer a UnicodeString containing the same elements as the receiver."
+	"cdemers 12/7/2016 Added for Unicode support."
+
+	^UnicodeString fromAddress: self yourAddress length: self size / 2! !
```

Comment:
I guess we should really rename UnicodeString. It isn't really a UnicodeString, rather a WideString. Conversions to/from bytes need to respect an encoding as well, so this would need to be more sophisticated eventually.